package.json

```diff
@@ -1,23 +1,33 @@
 {
   "name": "storybook-source-link",
-  "version": "4.0.0",
+  "version": "5.0.0",
   "description": "Provides a link to the story's source in the toolbar",
   "keywords": [
-    "source",
-    "link",
+    "storybook-addons",
     "storybook",
-    "storybook-addons"
+    "code",
+    "source",
+    "link"
   ],
-  "homepage": "https://storybook.js.org/addons/storybook-source-link/",
   "repository": {
     "type": "git",
     "url": "https://github.com/Sirrine-Jonathan/storybook-source-link"
   },
+  "type": "module",
   "author": "Jonathan Sirrine <sirrineprogramming@gmail.com>",
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/ts/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
+    "./preview": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/preview.js",
+      "require": "./dist/preview.cjs"
+    }
+  },
```

Comment:
Now that Webpack 4 is gone, this is the recommended way to expose exports, used by the addon-kit.